recipes/array-record/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "array-record" %}
+{% set version = "0.8.0a1" %}
```

Comment:
Normally we don't allow alpha tags on here, but I'm somewhat inclined to grant an exception here because this a bazel project and we're lucky it even has tags at all.  Any thoughts on this from the rest of @conda-forge/staged-recipes?